docs/swagger.yaml

```diff
@@ -341,6 +398,76 @@ components:
           example: 'https://example.com/myProduct'
       required:
         - redirectUrl
+    MerchantCallbackPostRequest:
+      title: MerchantCallbackPostRequest
+      x-stoplight:
+        id: bs090aq43jw2r
+      type: object
+      x-examples:
+        Example 1:
+          merchantQrId: ExampleStoreQR1
+          imageFormat: SVG
+          label: Example store - Kasse 1
+      properties:
+        merchantQrId:
+          type: string
```

Comment:
You should test for the required works if type in not string or null.

Reply:
It works with string